hypothesis-python/tests/common/debug.py

```diff
@@ -76,7 +77,7 @@ def find_any(
     settings = Settings(
         settings,
         max_examples=10000,
-        max_shrinks=0000,
+        phases=tuple(Phase)[:Phase.shrink],
```

Comment:
I'm not a huge fan of that way of writing this. It's pretty fragile against adding new phases. Why not `set(Phase) - {Phase.shrink}`?